boards/we/orthosie1ev/doc/index.rst

```diff
@@ -0,0 +1,251 @@
+.. zephyr:board:: we_orthosie1ev
+
+Overview
+********
+
+Orthosie-I-EV is an entry-level development board based on Orthosie-I,
+a module named for its small size. This board integrates complete Wi-Fi and Bluetooth® Low Energy functions.
+For more information, check `Orthosie-I Website`_.
+
+Hardware
+********
+
+ESP32-C3 is a single-core Wi-Fi and Bluetooth 5 (LE) microcontroller SoC,
+based on the open-source RISC-V architecture. It strikes the right balance of power,
+I/O capabilities and security, thus offering the optimal cost-effective
+solution for connected devices.
+The availability of Wi-Fi and Bluetooth 5 (LE) connectivity not only makes the device configuration easy,
+but it also facilitates a variety of use-cases based on dual connectivity.
+
+The features include the following:
+
+- 32-bit core RISC-V microcontroller with a maximum clock speed of 160 MHz
+- 400 KB of internal RAM
+- 802.11b/g/n/e/i
+- A Bluetooth LE subsystem that supports features of Bluetooth 5 and Bluetooth Mesh
+- Various peripherals:
+
+  - 12-bit ADC with up to 6 channels
+  - TWAI compatible with CAN bus 2.0
+  - Temperature sensor
+  - 3x SPI
+  - 1x I2S
+  - 1x I2C
+  - 2x UART
+  - LED PWM with up to 6 channels
+
+- Cryptographic hardware acceleration (RNG, ECC, RSA, SHA-2, AES)
+
+For more information, check the datasheet at `ESP32-C3 Datasheet`_ or the technical reference
+manual at `ESP32-C3 Technical Reference Manual`_.
+
+Supported Features
+==================
+
+Current Zephyr's Orthosie-I-EV board supports the following features:
```

Comment:
replace board name with the name used when running west build, with \`\` around it and refer to it as board target